test/spec/swagger/schema.js

```diff
@@ -525,3 +525,130 @@ test('support "const" keyword', async t => {
     }
   })
 })
+
+test('support "exposeHeadRoute" option', async (t) => {
+  const optWithoutHead = {
+    schema: {
+      response: {
```

Comment:
I would expect the `operationId` property here

Reply:
I rewrote the tests, can you please have another look?